docs/variables/accountage.md

```diff
@@ -0,0 +1,33 @@
+---
+id: accountage
+---
+
+# $(accountage)
+
+Returns the time that has passed since a user created their **Twitch account**.
+
+#### Parameters
+
+This variable takes **one** *optional* parameter that is a Twitch username of who to fetch the account age of. Defaults to the sender's username if not provided.
```

Comment:
This indicates the possession of a *single* **sender**, requiring you to add `'s` at the end making it `sender's`.

This would be different if it would indicate the possession of *multiple* **senders** *(ending with an `s`)*, requiring you to either add `'s` at the end making it `senders's`, or only `'` making it `senders'` like you did before. This is totally up to preference, both `'s` and `'` are correct here, but `'` is more widely "accepted".

This same rule also applies to all nouns ending with an **s**.

For example, both of the following are correct while the first one is more widely "accepted".

* `Brussels' population is an estimated 2,096,000 people.`
* `Brussels's population is an estimated 2,096,000 people.`